examples/api/lib/widgets/heroes/hero.1.dart

```diff
@@ -0,0 +1,103 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for Hero
+
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+
+void main() {
+  // Slow down time to see Hero flight animation.
+  timeDilation = 15.0;
+  runApp(const HeroApp());
+}
+
+class HeroApp extends StatelessWidget {
+  const HeroApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(title: const Text('Hero Sample')),
+        body: const Center(
+          child: HeroExample(),
+        ),
+      ),
+    );
+  }
+}
+
+class HeroExample extends StatelessWidget {
+  const HeroExample({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        ListTile(
+          leading: Hero(
+            tag: 'hero-default-tween',
+            child: _blueRectangle(size: 50.0, color: Colors.red[700]!.withOpacity(0.5)),
+          ),
+          onTap: () => _gotoDetailsPage(context),
+          title: const Text(
+            'Tap on the icon to view hero flight animation with default rect tween',
```

Comment:
Should this maybe say: "This red icon will use a default rect tween during the hero flight"

The other one would say: "This blue icon will use a custom rect tween during the hero flight"

... and then there is one button below them both to trigger the page transition?

I find it a little confusing that it says "Tap ... to view .. default/custom rect", but when you actually tap you see both at once...